README.md

```diff
@@ -63,6 +63,8 @@ Head over to the [Manage Resources In Multiple AWS Accounts](https://aws-control
 
 ### 6.0 Adopt Resources
 
+> :warning: For more information on migrating existing resources from the old [SageMaker Operators for Kubernetes](https://github.com/aws/amazon-sagemaker-operator-for-k8s) to the SageMaker ACK controller, see [Migrate resources to the new SageMaker Operators for Kubernetes](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-operators-migrate.html) in the *Amazon SageMaker Developer Guide*.
```

Comment:
lts create a separate section after this named migrating resources from old sagemaker operator

also do you mind modifying the adopt resources section introduction a bit to start with reference to https://aws-controllers-k8s.github.io/community/docs/user-docs/adopted-resource/.  this section was created because there was no documentation on the website, website docs are comprehensive
 the rest of the content here is just a sample for endpoint.